hpc/app/playbooks/workflows/start.yml

```diff
@@ -9,4 +9,14 @@
 
 - name: Start job in any compute
   import_playbook: start_batch.yml
-  when: scheduler == "batch"
\ No newline at end of file
+  when: scheduler == "batch"
+
+- name: Registering job_id in HPC Exporter
```

Comment:
This task should be exclusive to torque/slurm schedulers, since the HPC exporter does not yet monitor shell scripts, i.e. the scripts submitted without any workload managers.